docs/oid-info.md

```diff
@@ -182,7 +182,7 @@ that Sigstore operates.
 
 | GitHub [(docs)][github-oidc-doc] | GitLab | CircleCI | Buildkite | Fulcio Certificate Extension | Why / Notes / Questions                                                                                                                                                                |
```

Comment:
```suggestion
| GitHub [(docs)][github-oidc-doc] | GitLab [(docs)][https://docs.gitlab.com/ee/ci/cloud_services/#how-it-works] | CircleCI | Buildkite | Fulcio Certificate Extension | Why / Notes / Questions                                                                                                                                                                |
```

Seems like we have the claims documented in two places, not sure which one is better:
- https://docs.gitlab.com/ee/ci/cloud_services/#how-it-works
- https://docs.gitlab.com/ee/ci/secrets/id_token_authentication.html#token-payload